src/API/Run.hs

```diff
@@ -10,49 +10,80 @@ Holds the routines for parsing and interpreting a BoGL Prelude and Game file.
 
 module API.Run (_runCodeWithCommands) where
 
-import API.JSONData
+import API.JSONData (SpielResponse(..), SpielCommand(..), SpielResponses, spielParseError)
```

Comment:
This file still needs quite a bit of cleaning up, but I had to let it go eventually. We can do some more work to polish the API later and do it then.

Reply:
It would be good to clean things up once we've taken care of whatever comes up at this upcoming meeting. In addition, it would probably be good to make several smaller changes in separate PRs. Everything that is changed there will go live, and the more we change the more likely we are to introduce a bug. Small changes will make that easier to track down when they occur.